submissions/bmukha/hooli-style-popup/index.html

```diff
@@ -0,0 +1,205 @@
+              type="checkbox"
+              tabindex="4"
+            />
+            <div class="menu__wrapper">
+              <nav class="menu__wrapper__main">
+                <ul class="menu__items__main">
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/mail.png" width="48" alt="mail"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/files.png"
+                        width="48"
+                        alt="files"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/calendar.png"
+                        width="48"
+                        alt="calendar"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/documents.png"
+                        width="48"
+                        alt="documents"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/maps.png" width="48" alt="maps"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/chat.png" width="48" alt="chat"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img src="./popup-icons/news.png" width="48" alt="news"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/contacts.png"
+                        width="48"
+                        alt="contacts"
+                    /></a>
+                  </li>
+                  <li class="header__nav__item__main">
+                    <a href="#"
+                      ><img
+                        src="./popup-icons/classroom.png"
+                        width="48"
+                        alt="classroom"
+                    /></a>
+                  </li>
+                </ul>
+                <input
+                  type="checkbox"
+                  name="more_checkbox"
+                  class="more__checkbox"
+                  id="more__checkbox"
+                />
+                <label
+                  for="more__checkbox"
+                  class="more__label"
+                  tabindex="5"
```

Comment:
`tabindex` is not necessary in this task, and in your case, it cause problems - when you navigate with tab, focus should appear on icons first, and then on `More` button. 
Also when focus on `More` , hidden menu is not opens by clicking space on keyboard.